internal/cache/metadata/stat_cache.go

```diff
@@ -203,3 +219,46 @@ func (sc *statCacheBucketView) LookUp(
 
 	return
 }
+
```

Comment:
Size()  function will require to add size of e.f.

Not sure if you want to make changes in next PR for these